core/src/main/java/pl/allegro/tech/embeddedelasticsearch/IndexSettings.java

```diff
@@ -19,26 +19,23 @@
 
     private final List<TypeWithMapping> types;
     private final Optional<JsonNode> settings;
+    private final Optional<JsonNode> aliases;
 
     public static Builder builder() {
         return new Builder();
     }
 
-    public IndexSettings(List<TypeWithMapping> types, Optional<String> settings) {
+    public IndexSettings(List<TypeWithMapping> types, Optional<String> settings, Optional<String> aliases) {
```

Comment:
Unfortunately this constructor wasn't private from the beginning, so we have to assume that someone might used it instead of builder. Please provide additional constructor with default value for aliases

Reply:
I added new private constructor and leave the old as it was